internal/cmd/cluster/cluster.go

```diff
@@ -47,3 +49,30 @@ func newClusterHealthCmd(c *config.Cluster) *cobra.Command {
 	}
 	return healthCmd
 }
+
+func newClusterCatNodesCmd(c *config.Cluster) *cobra.Command {
```

Comment:
hey cat commands are diff section altogether. 

So what if we have `esctl cat nodes` instead ?